ebmbot/slack.py

```diff
@@ -45,11 +47,42 @@ def notify_slack(
             attempts += 1
             sleep(1)
             error = err
+    # The message has failed to post.
+    # Modify the Slack message to report the error and try posting it to the channel
+    COULD_NOT_NOTIFY_SLACK = "Could not notify slack"
```

Comment:
very minor nit: change to lower case since it's not a global anymore.
Could be header_text for consistency with message_text?